frontend/lib/types/TupleType.cpp

```diff
@@ -100,8 +100,8 @@ TupleType::getTupleType(Context* context, const TupleType* instantiatedFrom,
   QUERY_BEGIN(getTupleType, context, instantiatedFrom, subs,
                             isVarArgTuple);
 
-  auto name = UniqueString::get(context, "_tuple");
   auto id = parsing::getSymbolFromTopLevelModule(context, "ChapelTuple", "_tuple");
+  auto name = id.symbolName(context);
```

Comment:
Agh, that's too bad. The unenforced dependency bothers me a lot. Since we want both the id and name in 90% of the uses of this function, I'm thinking it makes more sense to just adjust it directly rather than introducing another helper.